src/player.cpp

```diff
@@ -3777,11 +3780,11 @@ void player::disp_status(WINDOW *w, WINDOW *w2)
             col_time = c_dkgray_red;
         }
     }
-    wprintz(w, col_time, " %3d", movecounter);
+    wprintz(w, col_time, " %d", movecounter);
 
-    wprintz(w, c_white, " %5s", _(move_mode.c_str()));
+    wprintz(w, c_white, " %s", _(move_mode == "walk" ? "W" : "R"));
```

Comment:
Translation will most likely not work here (did it work before). (If you tested this and it works, ignore the comment.) You'll have to add the translation marks to both strings ("W", "R") separately, otherwise they won't get extracted (and without extraction they won't be available for translation).

Btw. why is the "Stm" below (and above) not translated?